app/managers.py

```diff
@@ -0,0 +1,39 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name,)
+        )
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        format_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [Actor(*row) for row in format_cursor]
+
+    def update(self, id_to_update: int, first_name: str, lt_nm: str) -> None:

```

Comment:
The parameter name 'lt_nm' should be changed to 'last_name' to match the database column name and the method signature in `main.py`.